internal/testutil/mocksvc/deployment_service.go

```diff
@@ -0,0 +1,71 @@
+// Code generated by mockery. DO NOT EDIT.
+
+package mocksvc
```

Comment:
generated in mocksvc package so it doesn't conflict with testify mock package

Reply:
why do we have two mock packages?

Reply:
the Testify Mock package is the package with utilities in the Testify library.  mocksvc is the package I've chosen to create the mocked interfaces so you can use both package in a test file withouth a conflict and without needing to create an alias for one of the packages